conf/supervisord.conf

```diff
@@ -5,7 +5,7 @@ logfile=/dev/null
 logfile_maxbytes=0
 
 [program:web]
-command=newrelic-admin run-program gunicorn checkmate.app:create_app() --bind 0.0.0.0:9099
```

Comment:
The `checkmate.app:create_app()` bit now comes from the `use` setting in `production.ini` as it does for our other apps (and as it already did for Checkmate itself in dev too)